akka-remote/src/main/scala/akka/remote/artery/AeronSink.scala

```diff
@@ -65,33 +106,22 @@ class AeronSink(channel: String, aeron: Aeron) extends GraphStage[SinkShape[Aero
         // FIXME handle Publication.CLOSED
         // TODO the backoff strategy should be measured and tuned
         if (result < 0) {
-          if (backoffCount == 1) {
-            println(s"# drop") // FIXME
-            pull(in) // drop it
-          } else if (backoffCount <= 15) {
-            // TODO Instead of using the scheduler we should handoff the task of
-            // retrying/polling to a separate thread that performs the polling for
-            // all sources/sinks and notifies back when there is some news.
-            backoffCount -= 1
-            if (backoffCount <= 10)
-              scheduleOnce(Backoff, 50.millis)
-            else
-              scheduleOnce(Backoff, 1.millis)
-          } else {
+          backoffCount -= 1
+          if (backoffCount > 0) {
             idleStrategy.idle()
```

Comment:
do we really need the idleStrategy anymore? I would expect a short amount of spinning here (~10-20 iterations) and then delegating immediately to an external thread to avoid starvation issues.